### PR TITLE
dist: updates and documentation

### PR DESCRIPTION
**Pull request by illwieckz**
Minor updates, beware that variable names in `paths.conf` were modified.

Also provide some documentation about those files.

### CODE REVIEW

**Comment by bmorel** [comment]
While at it, I've seen that server.cfg still uses "alpha" in the description.

**Review by necessarily-equal — APPROVED** [review]
I have some comments that would be nice to address, but they aren't really about this PR so you may want to merge this before addressing them

File: dist/scripts/unvanquished [hunk 1]
```diff
@@ -63,7 +74,7 @@ then
 	cat <<-EOF
 	${program_name}: run ${program_desc}
 
-	Usage: ${program_name} [option]
+	Usage: ${program_name} [options] [engine and game options]
 
 	Options:
 	${tab}--help          print this help
```
[the remark this answers is not in the document]
Comment (illwieckz) [reply dist/scripts/unvanquished:80]: > I'm just thinking that this hides the daemon help.

Good point!

Maybe I can just make it call `daemon --help` so the script help also displays daemon help.

File: dist/scripts/unvanquished [hunk 2]
```diff
@@ -73,47 +84,44 @@ then
 	${tab}--system        use system Unvanquished home directory
 	${tab}--gdb           run with GDB debugger
```
[the remark this answers is not in the document]
Comment (illwieckz) [reply dist/scripts/unvanquished:85]: Welp, if I'm right this what what was doing a previous script, so when I rewrote it from scratch I also implemented that to not regress. But I'm unhappy with it. We may just expects the user to set a DEBUG variable.

For example:

```sh
DEBUG=`gdb -args` unvanquished
```

```sh
DEBUG=`valgrind` unvanquished
```

[the remark this answers is not in the document]
Comment (necessarily-equal) [reply dist/scripts/unvanquished:85]: or `WRAPPER=…` ? or --wrapper="gdb --args" ?

File: dist/scripts/unvanquished (hunk 1 above)
[the remark this answers is not in the document]
Comment (necessarily-equal) [reply dist/scripts/unvanquished:80]: Maybe `exec daemon --help` would help avoiding https://github.com/DaemonEngine/Daemon/issues/631.

**Comment by bmorel** [comment]
LGTM.

As for the wrappers... I have doubts about gdb's usefulness (through a script...) but calling i.e. valgrind through a script really feels stupid to me. People who know how to use advanced tools such as valgrind will anyway want to do their own command line (especially valgrind, that one will have to tweak to get the kind of infos really wished for).

File: dist/scripts/unvanquished (hunk 1 above)
[the remark this answers is not in the document]
Comment (illwieckz) [reply dist/scripts/unvanquished:91]: This is what I've done, I may also have fixes (or workarounds) for https://github.com/DaemonEngine/Daemon/issues/631 this but there is no urge.

File: dist/scripts/unvanquished [hunk 3]
```diff
@@ -73,47 +84,44 @@ then
 	${tab}--system        use system Unvanquished home directory
```
[the remark this answers is not in the document]
Comment (illwieckz) [reply dist/scripts/unvanquished:95]: I'll postpone that question for later (I have no strong opinion and that should not be blocking).

File: dist/scripts/unvanquished (hunk 2 above)
[the remark this answers is not in the document]
Comment (illwieckz) [reply dist/scripts/unvanquished:96]: I'll postpone that question for later as well (we can do better but there is no regression).

**Pull request merged**